k2/csrc/fsa_equivalent_test.cc

```diff
@@ -0,0 +1,86 @@
+// k2/csrc/fsa_equivalent_test.cc
+
+// Copyright (c)  2020  Haowen Qiu
+
+// See ../../LICENSE for clarification regarding multiple authors
+
+#include "k2/csrc/fsa_equivalent.h"
+
+#include <algorithm>
+#include <utility>
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "k2/csrc/fsa.h"
+
+namespace k2 {
+
+TEST(Properties, RandomPathFail) {
```

Comment:
Forgot to mention that it should NOT be `Properties` here.

Reply:
updated in this pr https://github.com/danpovey/k2/pull/17